build/build.py

```diff
@@ -401,6 +401,11 @@ async def main():
     bazel_command_base.append(
         f"--repo_env=HERMETIC_PYTHON_VERSION={args.python_version}"
     )
+    # Let's interpret X.YY-ft version as free-threading python and set rules_python config flag:
```

Comment:
@hawkinsp what do you think about this?

Reply:
Seems perfectly reasonable to me. If we want to, we can change this later.